rfcs/0005-pablo-distribution-assets/0005-pablo-distribution.adoc

```diff
@@ -0,0 +1,632 @@
+= Design Proposal: Pablo Fees & Staking Rewards Distribution
+:math:
+:stem:
+:imagesoutdir: images
+:imagesdir: images
+:toc:
+:toclevels: 4
+:sectnums:
+:sectnumlevels: 4
+
+== Abstract
+
+This document proposes the Pablo distribution(token and pool trading fees) mechanism while considering various options and capturing the discussions about the subject.
+
+`TODO summarise the mechanism`
```

Comment:
it seems summary is what we need first hand. because need to have clear team goals

Reply:
This is actually the opposite of how normal technical documentation process work, you write the abstract at the end. 
Order roughly is as follows,
1. Background
2. Requirements
3. Method
4. Implementation/Milestones etc
5. Abstract

Reply:
please summarize in PR, not in comment